frontend/src/data/utils/Constants.ts

```diff
@@ -1,6 +1,8 @@
 // Constant strings to be used in tests and elsewhere
 
 // TODO add values for the other categories. Maybe convert to an enum.
+export const ALL = "All";
+export const ALL_OF_KNOWN = "All of known";
```

Comment:
Doesn't seem like this is used anywhere?

Reply:
good catch!